src/client/apolloClient.ts

```diff
@@ -62,6 +63,20 @@ export const apolloClient = (() => {
         InProgressReferral: {
           keyFields: (referral) => referral.name as string,
         },
+        QuoteBundle: {
+          keyFields: (quoteBundle) => {
+            const quotes = (quoteBundle as QuoteBundle).quotes ?? []
+
+            if (quotes.length === 0) return
+
+            // QuoteBundle:<id>+<id>
+            return quotes.reduce(
+              (id, quote, index) =>
+                (id += `${quote.id}${index < quotes.length - 1 ? '+' : ''}`),
+              'QuoteBundle:',
```

Comment:
hehe isn't this overly complicated?

```suggestion
            // QuoteBundle:<id>+<id>
            return `QuoteBundle:${quotes.map((quote) => quote.id).join('+')}`
```

Reply:
OMG hahhaha
So much better